examples/proc_FIR.py

```diff
@@ -67,51 +75,72 @@
                 fl.next("after auto-clock correction")
                 fl.image(s)
             # }}}
-        np.squeeze(s)
-        s, _ = prscr.rough_table_of_integrals(s, fl=fl)
+        s.squeeze()
+        s, ax_last = prscr.rough_table_of_integrals(s, fl=fl)
+        # TODO ☐: I'm assuming this is copied from previous.  I think that this
+        # is probably something we want to address in rough_table_of_integrals
+        # (if we have multiple scans, we want the result to be signal
+        # averaged).  Likely you would signal average after alignment, but
+        # before you actually take the integral over range of frequencies.
+        # But, I think it's useful for you to think about this as well and
+        # implement a solution.
         if "nScans" in s.dimlabels:
-            s_int = s.C.mean("nScans")
-        else:
-            s_int = s.C
-        print(s_int)
-        s.set_plot_color_next()
+            s.mean("nScans")
+        # TODO ☐: below, you are tearing apart the nddata object and putting it
+        # back together -- that's always a bad sign
```

Comment:
this comment I largely take care of and break down next steps below -- but this is a good rule of thumb